src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,25 @@
+package tests;
+
+import object.user.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import testResultUtil.TestResultExtension;
+import utils.singleton.SingletonInstance;
+@ExtendWith(TestResultExtension.class)
+
+public class BaseTest {
+    public static User validUser;
+    public static User invalidUser;
```

Comment:
What about not making them public and static ?  Can it be protected and without static ?